common/exf.c

```diff
@@ -457,7 +457,7 @@ file_spath(SCR *sp, FREF *frp, struct stat *sbp, int *existsp)
 
 	/*
 	 * If the name is NULL or an explicit reference (i.e., the first
-	 * component is . or ..) ignore the O_PATH option.
+	 * component is . or ..) ignore the O_PATH__ option.
```

Comment:
Maybe just `#undef O_PATH`. This symbol appears earlier than Linux. I would like to keep it for cross-reference purposes.